satpy/writers/geotiff.py

```diff
@@ -147,6 +148,25 @@ def save_image(self, img, filename=None, dtype=None, fill_value=None,
                 the index range of the palette
                 (ex. `cmap.set_range(0, len(colors))`).
             tags (dict): Extra metadata to store in geotiff.
+            overviews (list): The reduction factors of the overviews to include
+                in the image, eg::
+
+                    scn.save_datasets(overviews=[2, 4, 8, 16])
+
+                If provided as an empty list, then levels will be
+                computed as powers of two until the last level has less
+                pixels than `overviews_minsize`.
+                Default is to not add overviews.
+            overviews_minsize (int): Minimum number of pixels for the smallest
+                overview size generated when `overviews` is auto-generated.
+                Defaults to 256.
+            overviews_resampling (str): Resampling method
+                to use when generating overviews. This must be the name of an
+                enum value from :class:`rasterio.enums.Resampling` and
+                only takes effect if the `overviews` keyword argument is
+                provided. Common values include `nearest` (default),
+                `bilinear`, `average`, and many others. See the rasterio
+                documentation for more information.
```

Comment:
The order of the arguments in the docstring doesn't match the one in the function signature, can you fix this?

Reply:
Done. Will do the same to the trollimage PR.